src/views/TradingCompetition/components/Countdown/TimerTooltip.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
```

Comment:
A PR has been made exclusively for this component & the Timer - https://github.com/pancakeswap/pancake-frontend/pull/772